### ISSUE TITLE
Add Interactive Highlights to BASE

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#514]
Discussed with respect to #513, adding Interactive Highlights will likely be very quick and present an immediate solution for that issue. Would also be great to get another feature in a sim.

**Comment by jessegreenberg** [comment]
Added in the above commits. Will be following up in https://github.com/phetsims/balloons-and-static-electricity/issues/513, closing.

**Comment by jessegreenberg** [comment]
Actually, reopening for review.

@terracoda can you please verify the Interactive Highlights in BASE and confirm that they are working as expected and well enough for publication?

**Comment by jessegreenberg** [comment]
Link to version with this: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-dev.33/phet/balloons-and-static-electricity_en_phet.html

**Comment by terracoda** [comment]
When I click the balloon, it becomes grabbed, can you make the pink highlight dotted in that case?

Ungrabbed state = hover and focus = solid pink highlight
Grabbed state = clicked and pressed or grabbed with keyboard = dotted pink highlight

Is that possible? It would be more aligned with our idea of a grabbed (pressed) state of the balloon.

I realize it may not be possible in quick order...not sure.

**Comment by jessegreenberg** [comment]
It should be possible, but this is also blocked by the issue mentioned in https://github.com/phetsims/friction/issues/200.

**Comment by terracoda** [comment]
I can temporarily live with the solid interactive highlight for the balloons if phetsims/friction200 cannot be addressed before BASE is ready to be published with its preferences.

If that happens we can create a new issue or mark this one for a maintenance release with Friction's issue.

This will be an issue for Faraday's Law and possibly for RaP as well.

**Comment by terracoda** [comment]
@jessegreenberg, I confirmed with Emily that leaving the interactive highlight as-is as a solid pink line is not a blocking issue for BASE.

The interactive highlights have been verified, so should I close this issue?
And then create a new issue for a maintenance release, if needed?

**Comment by jessegreenberg** [comment]
@terracoda that sounds good, lets make a new issue to improve the highlights but close this one. Thanks for the update and for reviewing.
